LayoutTests/http/tests/site-isolation/notify-done-expected.txt

```diff
@@ -0,0 +1,5 @@
+
+--------
+Frame: '<!--frame2-->'
+--------
+I am a simple document that calls notifyDone when loaded.
```

Comment:
This gets the output from the wrong process.  The whole point is that if we're not in the main frame process we need to send a message to the main frame process and get output from that process.